website/src/components/views/analyzeSingleVariant/SelectVariant.astro

```diff
@@ -0,0 +1,14 @@
+---
+import { PageHeadline } from '../../../styles/containers/PageHeadline';
+---
+
+<div class='mt-20 flex min-h-64 flex-col items-center'>
+    <PageHeadline>Analyze a single variant</PageHeadline>
+    <div class='max-w-xl'>
+        <p class='mb-2'>To proceed, please select a variant filter.</p>
+        <p>
+            Note that the dataset filter is applied to the entire dataset. Adjust the dataset filter as needed to refine
```

Comment:
![grafik](https://github.com/user-attachments/assets/5a3afd3a-5f32-420e-b931-270c617e035b)

In the filter panel on the left, it's called "Filter dataset", but in the text we refer to it as "dataset filter". IMO we should make that consistent. I'd prefer changing it in the filter panel on the left, since "Dataset Filter" would match "Variant Filter" below better.

Reply:
I changed it to "Dataset Filter", which we will also need in #352 and #351